src/service/runnerService.ts

```diff
@@ -1,10 +1,61 @@
 import { resolve as res } from 'path';
 import * as vscode from 'vscode';
+import { CodeUtil } from '../common/codeUtil';
 import { FileManager, FileModel } from '../common/fileManager';
 import { ConfigKey, Global } from '../common/global';
 import { Process } from '../common/processWrapper';
 
 export class RunnerService {
+    /** Align assignments and comments in selection. */
+    public static async alignSelection(): Promise<void> {
```

Comment:
This definitely seems to work as expected, but what happens when the user re-formats the file? This PR should update the formatter to not revert these alignments... I'm not sure how to do that though :(

Reply:
How about add special directive for formatter:
```autohotkey
;@AHK++-FormatRemoveExtraSpaceOff
a     := "a"
abc := "abc"
;@AHK++-FormatRemoveExtraSpaceOn
OR
;@AHK++-Format (to disable all possible applied directives)
```

Reply:
Let's go with this, closer to the first one but more specific to the feature:

```ahk
;@AHK++AlignAssignmentOn
a   := "a"
abc := "abc"
;@AHK++AlignAssignmentOff
```

It's not necessary for this PR, but it'd be awesome to have the formatter automatically align assignments based on these directives as well. If there is no `off`, we just align everything below the `on` through the end of the file.

If it can't be done in this PR, please open an issue for this :)